tasks/rvm.yml

```diff
@@ -36,6 +36,16 @@
   shell: curl -sSL https://rvm.io/mpapis.asc | gpg --import -
   when: not ansible_check_mode and rvm1_gpg_keys != '' and gpg_result.rc != 0
 
+- name: Install which (CentOS, RHEL)
+  become: yes
```

Comment:
remove `become` and `become_user`, these entries are declared on the [main.yml](https://github.com/rvm/rvm1-ansible/blob/master/tasks/main.yml)

Reply:
`become` is definitely redundant. With regard to `become_user`, it is not clear that the `rvm1_user` will have the permissions to install the package. I can certainly try to remove both of them and see if Travis validates.